incubator/hnc/Makefile

```diff
@@ -48,20 +48,20 @@ test: build
 # Builds all binaries (manager and kubectl) and manifests
 build: generate fmt vet manifests
 	go build -o bin/manager ./cmd/manager/main.go
-	go build -o bin/kubectl/kubectl-hierarchical_namespaces ./cmd/kubectl/main.go
+	go build -o bin/kubectl/kubectl-hns ./cmd/kubectl/main.go
 
 # Clean all binaries (manager and kubectl)
 clean: krew-uninstall
 	-rm -rf bin/*
 	-rm -rf manifests/*
-	-rm -f ${GOPATH}/bin/kubectl-hierarchical_namespaces
+	-rm -f ${GOPATH}/bin/kubectl-hns
 	-rm -f ${GOPATH}/bin/kubectl-hns
 
 # Install kubectl plugin
 kubectl: build
-	cp bin/kubectl/kubectl-hierarchical_namespaces ${GOPATH}/bin/kubectl-hierarchical_namespaces
-	ln -fs ${GOPATH}/bin/kubectl-hierarchical_namespaces ${GOPATH}/bin/kubectl-hns
-	@echo "Installed kubectl-hierarchical_namespaces and kubectl-hns to GOPATH/bin"
+	cp bin/kubectl/kubectl-hns ${GOPATH}/bin/kubectl-hns
+	ln -fs ${GOPATH}/bin/kubectl-hns ${GOPATH}/bin/kubectl-hns
```

Comment:
Did you run this? What happened? It should be redundant, at any rate :)

Reply:
Yeah I ran through the make build and ran a few commands to make sure things were still working. I think it must have nooped or gave a failure that didn't stop the build. You know what's funny is that I had deleted the lines but the Makefile didn't get saved. I'll be a little bit more careful next time.